test/tests/tests.vcxproj

```diff
@@ -66,6 +66,7 @@
     <ClCompile Include="Localisation.cpp" />
     <ClCompile Include="MultiLaunch.cpp" />
     <ClCompile Include="ReplayTests.cpp" />
+    <ClCompile Include="PlayTests.cpp" />
```

Comment:
If this is merged we probably want a more meaningful name so we avoid people from always adding to the same test? Or do we want the big file with tests?

Reply:
I was thinking that this can start as a single file with the tests names being the descritive part (just like ReplayTests.cpp is not descriptive to what the test is verifying). If this turns out to grow too much, then it coulde be rethinked.